source/imports/value-objects/nin.js

```diff
@@ -0,0 +1,10 @@
+const NIN = Space.domain.ValueObject.extend('NIN', {
```

Comment:
what does `NIN` mean? Google does not tell me anything useful apart from a croatian city and Nine inch nails :smile: